README.md

```diff
@@ -50,21 +57,26 @@ The choices are `border`, `outline`, `padding`, and `margin`. You will leave two
 Your Answer:
 
 ```text
-___: Defines the distance between an element's border and adjacent elements' borders.
+margin: Defines the distance between an element's      border and adjacent elements' borders.
+
 ___: Inserts a "wall" around an element.
-___: Defines the distance between an element's content and its border.
-___: Defines the width of an element.
-___: Overlays a "wall" on top of an element.
+
+padding: Defines the distance between an element's content and its border.
+
+_border__: Defines the width of an element.
```

Comment:
`width` defines the width of an element. `border` inserts a "wall" around an element.